ExampleAppUITests/TradeItSDK2UITestsPortfolioFlow.swift

```diff
@@ -123,63 +123,4 @@ class TradeItSDK2UITestsPortfolioFlow: XCTestCase {
         XCTAssertFalse(app.tables.staticTexts["Margin Acc**cct4"].exists)
         XCTAssertFalse(app.tables.staticTexts["OPTIONS SU**cct5"].exists)
     }
-    
```

Comment:
These have been commented for a while. Seeing as I'm stripping out other FX code for now it seems the right time to remove this too.